app/models/pet.rb

```diff
@@ -32,9 +34,12 @@ class Pet < ApplicationRecord
 
   validates :name, presence: true
   validates :birth_date, presence: true
-  validates :size, presence: true
   validates :breed, presence: true
   validates :sex, presence: true
+  validates :weight_from, presence: true
+  validates :weight_to, presence: true
```

Comment:
Let's force them to be numerical in case of accidentally adding a string or none number item - https://guides.rubyonrails.org/active_record_validations.html#numericality

Like must be greater than 0 for example

Reply:
Yes good call. Done.